doc/api/repl.md

```diff
@@ -475,7 +547,25 @@ properties:
 
 The following example shows two new commands added to the REPL instance:
 
-```js
+```mjs
+import repl from 'node:repl';
+
+const replServer = repl.start({ prompt: '> ' });
+replServer.defineCommand('sayhello', {
+  help: 'Say hello',
+  action(name) {
+    this.clearBufferedCommand();
+    console.log(`Hello, ${name}!`);
```

Comment:
this output doesn't quite make sense - where would `${name}` ever come from?

Reply:
it's passed when you execute the `.sayhello` command, like so:

```sh
> .sayhello bnb
Hello, bnb!
```